task_processing/plugins/kubernetes/kubernetes_pod_executor.py

```diff
@@ -442,19 +444,38 @@ def _create_container_definition(
                 privileged=task_config.privileged,
             )
 
+        # Set any optional requests
+        requests: Dict[str, Union[str, float]] = {}
+        if 'request_cpus' in task_config and task_config.request_cpus:
+            requests['cpus'] = task_config.request_cpus
+        if 'request_memory' in task_config and task_config.request_memory:
+            requests['memory'] = f"{task_config.request_memory}Mi"
+        if 'request_disk' in task_config and task_config.request_disk:
+            requests['ephemeral-storage'] = f"{task_config.request_disk}Mi"
```

Comment:
imo, it'd be nice to keep the disk stuff as-is since I don't think we have much of a system reservation for disk (if we have any at all) - but if this is just for completeness, it should be fine to leave :)

Reply:
Its only for completeness. I can remove if you have strong feelings about this.

Reply:
yea, I say we can circle back and add this later if we ever end up needing this - it makes me a tad nervous as-is tho (mostly 'cause I'm not sure if the kubelet will protect us by evicting things when the node is under disk pressure or if we'll just run the node out of disk and kill it :p)